src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2693,6 +2693,47 @@ namespace UnitarySystems {
+                                                                0.0,
+                                                                this->m_MaxONOFFCyclesperHour,
+                                                                this->m_HPTimeConstant,
+                                                                this->m_FanDelayTime,
+                                                                DataHVACGlobals::CompressorOperation::Off,
+                                                                0.0,
+                                                                FirstHVACIteration);
+                state.dataSize->DataConstantUsedForSizing = WaterToAirHeatPumpSimple::GetCoilCapacity(
+                    state, DataHVACGlobals::cAllCoilTypes(this->m_HeatingCoilType_Num), this->m_HeatingCoilName, ErrFound);
+                EqSizing.DesHeatingLoad = state.dataSize->DataConstantUsedForSizing;
+                state.dataSize->DataFractionUsedForSizing = 1.0;
+                SizingMethod = DataHVACGlobals::AutoCalculateSizing;
+                this->m_DesignHeatingCapacity = DataSizing::AutoSize;
+                if (this->m_CoolingCoilType_Num == DataHVACGlobals::Coil_CoolingWaterToAirHPSimple)
+                    // adjusted cooling coil capacity
+                    WaterToAirHeatPumpSimple::SimWatertoAirHPSimple(state,
+                                                                    blankString,
+                                                                    this->m_CoolingCoilIndex,
+                                                                    this->m_CoolingCoilSensDemand,
+                                                                    this->m_CoolingCoilLatentDemand,
+                                                                    0.0,
+                                                                    0.0,
+                                                                    this->m_MaxONOFFCyclesperHour,
+                                                                    this->m_HPTimeConstant,
+                                                                    this->m_FanDelayTime,
+                                                                    DataHVACGlobals::CompressorOperation::Off,
+                                                                    0.0,
+                                                                    FirstHVACIteration);
+                state.dataSize->DataConstantUsedForSizing = WaterToAirHeatPumpSimple::GetCoilCapacity(
+                    state, DataHVACGlobals::cAllCoilTypes(this->m_CoolingCoilType_Num), this->m_CoolingCoilName, ErrFound);
+                EqSizing.DesCoolingLoad = state.dataSize->DataConstantUsedForSizing;
+                state.dataSize->DataFractionUsedForSizing = 1.0;
+                SizingMethod = DataHVACGlobals::AutoCalculateSizing;
+                this->m_DesignCoolingCapacity = DataSizing::AutoSize;
```

Comment:
Do you realize that UnitarySystem calls the new sizing functions (e.g., CoolingCapacitySizing) to choose capacity and air flow before a coil is sized? Have you tested this coil model in UnitarySystem? Be prepared to test this configuration before this can merge.